websites/S/Serendia Squad/dist/metadata.json

```diff
@@ -12,7 +12,7 @@
   "service": "Serendia Squad",
   "version": "1.0.0",
   "logo": "https://bbq.is-pretty.cool/5o6DS5b.png",
-  "thumbnail": "https://bbq.is-pretty.cool/5o6DS5b.png",
+  "thumbnail": "https://barbecue.is-serious.business/vtX7xgA.png",
```

Comment:
upload to imgur

Reply:
bro it's owo

Reply:
bruh this is [owo](https://whats-th.is/)